src/JsonService.ts

```diff
@@ -136,6 +144,7 @@ export class JsonService {
         if (basicAuth !== undefined) {
             headers["Authorization"] = "Basic " + basicAuth;
         }
+        this.appendExtraHeaders(headers);
```

Comment:
add a space between the if block and this new code

Reply:
Ok